tests/e2e/specs/schemas.spec.ts

```diff
@@ -1,238 +1,394 @@
-    test.beforeEach(async ({ page, electronApp }) => {
-      // Stub the dialog that tells you the connection was created
+    // delete the subject if it was created during the test
+    if (subjectName) {
+      // stub the system dialog (warning modal) that appears when hard-deleting
       await stubMultipleDialogs(electronApp, [
         {
           method: "showMessageBox",
           value: {
-            response: 0, // Simulates clicking "OK"
+            response: 0, // simulate clicking "Yes, Hard Delete" (first button)
             checkboxChecked: false,
           },
         },
       ]);
 
-      await page.getByLabel("Resources Section").hover();
-      await page.getByLabel("Add New Connection").click();
-      await page.getByLabel("Enter manually").locator("a").click();
-      const webview = page.locator("iframe").contentFrame().locator("iframe").contentFrame();
-
-      const { apiKey, apiSecret, endpoint } = extractConfluentCredentials();
-
-      await webview.locator("#name").click();
-      await page.waitForTimeout(100);
-      await page.keyboard.type("Playwright");
-
-      await webview.locator("#formconnectiontype").click();
-      await page.waitForTimeout(100);
-
-      await webview.locator("#formconnectiontype").selectOption("Confluent Cloud");
-
-      await webview.locator('[id="schema_registry\\.uri"]').click();
-      // Wait for the input to be focused.
-      await page.waitForTimeout(100);
-      await page.keyboard.type(endpoint);
-
-      await webview.locator('[id="schema_registry\\.auth_type"]').click();
-      await webview.locator('[id="schema_registry\\.auth_type"]').selectOption("API");
+      // replace this with right-click context actions once this issue is resolved:
+      // https://github.com/confluentinc/vscode/issues/1875
+      await executeVSCodeCommand(page, "Confluent: Delete All Schemas in Subject");
+
+      // select the subject to delete
+      const subjectInputBox = new InputBox(page);
+      await expect(subjectInputBox.placeholder).toBeVisible();
+      await subjectInputBox.input.fill(subjectName);
+      await subjectInputBox.confirm();
+
+      // select the Hard Delete option
+      const deletionQuickpick = new Quickpick(page);
+      const hardDelete = deletionQuickpick.items.filter({ hasText: "Hard Delete" });
+      await expect(hardDelete).not.toHaveCount(0);
+      await hardDelete.click();
+
+      // enter the confirmation text input
+      const confirmationBox = new InputBox(page);
+      await expect(confirmationBox.input).toBeVisible();
+      await confirmationBox.input.fill(deletionConfirmation);
+      await confirmationBox.confirm();
+
+      // the system dialog is automatically handled by the stub above, no need to handle it here
+
+      const deletionNotifications = notificationArea.infoNotifications.filter({
+        hasText: /hard deleted/,
+      });
+      await expect(deletionNotifications.first()).toBeVisible();
+    }
```

Comment:
TODO: fix this since it's currently broken when these tests run on Linux

Reply:
Fixed in https://github.com/confluentinc/vscode/pull/2135/commits/e7c08a694f382cb330e745032ea83860ee591c82